CHANGELOG.md

```diff
@@ -10,11 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]
 ------------
 
+[v0.8.0] - 2020-01-22
+---------------------
+
 ### Added
 
 - Support for backtracing in debug and release builds.
     - Implementations for GNU functions `backtrace` and `backtrace_symbols` (defined in execinfo.h)
     - Enclaves are built using `-fno-omit-frame-pointer` for accurate backtraces.
+- Support for custom attestation data formats via new plugin model.
```

Comment:
This probably requires more elaboration, can we link to their sample/docs on this? If they don't already, they need to add them, and if they do, we need to check that they are up-to-date with what is in the code.

Reply:
Added a link to design doc.